packages/safe-core-sdk/src/safeFactory/index.ts

```diff
@@ -5,12 +5,14 @@ import {
   SafeVersion,
   TransactionOptions
 } from '@gnosis.pm/safe-core-sdk-types'
+import abi from 'ethereumjs-abi'
```

Comment:
Both `ethers` and `web3` have utilities for abi encoding. Can we add corresponding functions to the adapters? Since the library can be used in-browser, we should be cautious about blowing the bundle size imo